src/scss/components/intro.scss

```diff
@@ -0,0 +1,57 @@
+.intro {
+  background-image: url('../static/assets/blue-form.png');
+  background-position-x: right;
+  background-position-y: 254px;
+  background-repeat: no-repeat;
+
+  &__container-general {
+    align-items: center;
+    background-image: url('../static/assets/orange-form.png');
+    background-position-x: 53px;
+    background-position-y: 14px;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__image--0 {
+    height: 334px;
+    margin-left: 99.3px;
+    margin-right: 29px;
+    margin-top: 38px;
+    width: 247px;
+  }
+
+  &__image--1 {
+    float: left;
+    height: 150px;
+    margin-left: 29px;
+    margin-right: 196px;
+    margin-top: 260px;
+    position: absolute;
+    width: 150px;
+  }
+
+  &__title {
+    font-family: $hkgrotesk-bold;
+    font-size: $mobile-title;
+    height: 90px;
+    margin-left: 198px;
+    margin-right: 53px;
+    margin-top: 24px;
+    width: 124px;
+  }
+
+  &__paragraph {
+    color: $gray;
+    font-family: $hkgrotesk-regular;
+    font-size: $mobile-paragraph;
+    height: 88px;
+    margin-left: 49px;
+    margin-right: 49px;
+    margin-top: 27px;
+    width: 270px;
+  }
+}
+
+
```

Comment:
Remove one of the extra lines in the end of this file

Reply:
Camilo, thanks for your check, the errors that you report me was already fixed.